.gitignore

```diff
@@ -39,3 +39,10 @@ buildNumber.properties
 .mvn/wrapper/maven-wrapper.jar
 .polyglot.*
 .settings
+jvm/.project
```

Comment:
Projects & Classpath files of Eclipse projects shall not be in the .gitignore :)